test/python/rest/test_rest_bluemix.py

```diff
@@ -15,3 +16,22 @@ def setUp(self):
         Tester.setup_streaming_analytics(self, force_remote_build=True)
         self.sc = StreamingAnalyticsConnection()
         self.test_config[ConfigParams.STREAMS_CONNECTION]=self.sc
+
+        self.is_v2 = False
```

Comment:
Is this used?

Reply:
Yes, in `primitives_caller`, we need to check the service version to determine whether the resourceAllocations should be retrieved from the job.